src/python/pants/task/testrunner_task_mixin.py

```diff
@@ -450,6 +451,31 @@ def chroot(self, targets, workdir):
         yield chroot
 
   def _execute(self, all_targets):
+
+    deprecated_conditional(
+      lambda: self.get_options().is_default("fast"),
+      removal_version="1.25.0.dev1",
+      entity_description="using the default value for --no-fast/--fast",
```

Comment:
I couldn't figure out how to get the option scope. I'd love to change this to say `--no-test-pytest-fast` or `--no-test-go-fast`, for example. 

Note that this is a mixin. At runtime, I was able to access `self.stable_name()` to get the value `'pants_backend_python_tasks_pytest_run_PytestRun'`, but couldn't find via playing around in PDB a way to get the actual scope.